control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -209,6 +214,17 @@ def test_service_exposes_read_only_record_and_reconciliation_inspection(self) ->
             tuple(record["reconciliation_id"] for record in status_view.records),
             ("reconciliation-001", "reconciliation-002"),
         )
+        self.assertNotIn(
+            "latest_native_payload",
+            status_view.records[0]["subject_linkage"],
+        )
+        self.assertNotIn(
+            "latest_native_payload",
+            status_view.records[1]["subject_linkage"],
+        )
+        stored_latest = service.get_record(ReconciliationRecord, "reconciliation-002")
```

Ruff (0.15.10)

[error] Undefined name `ReconciliationRecord`

(F821)